src/pages/ReferralAndCouponCodes/index.js

```diff
@@ -16,6 +16,13 @@ const useStyles = makeStyles((theme) => ({
   expanded: {
     flexGrow: 1,
   },
+  mainTitle: {
+    display: "flex",
+    marginTop: theme.spacing(2.1),
+    marginRight: theme.spacing(3),
+    alignItems: "center",
+    justifyContent: "space-between",
+  },
```

Comment:
Hey, @Abhijay007 I don't think making the cancel button align with the title is a good idea you can revert this change :)

Reply:
@adarsh-technocrat can you please elaborate more on why it is not a good idea to align the cancel button with the title, 
I think there is nothing wrong in aligning the cancel button with the title as it was already in the same `div` I just did some minor changes to that div by adding a separate class. And in all other pages of the site as well the title of the page is in `Appbar`  along with goBack and navigate button :)

Reply:
> @adarsh-technocrat can you please elaborate more on why it is not a good idea to align the cancel button with the title

oooh Okay! @Abhijay007 I think you are thinking that the title need's to be part of the `app Bar`  that's why you made that cancel button align to the title right! but actually, that was the purpose of the design that was not an alignment issue! 

Adding the Cancel button to the  `Material AppBar` itself makes it responsive so I don't think this will make any issue!
```javascript
 <AppBar position="static" className={classes.appBar}>
        <Toolbar variant="dense">
          <Box className={classes.expanded} />
          <IconButton
            edge="end"
            className={classes.clearIcon}
            color="inherit"
            onClick={() => history.goBack()}
          >
            <ClearIcon />
          </IconButton>
        </Toolbar>
      </AppBar>
```

Reply:
> And in all other pages of the site as well the title of the page is in `Appbar` along with goBack and navigate button :)

 @Abhijay007  Actually it's not a mandatory thing to sync every route page with the same format.  We can change the format for some routes like this section just the thing is it should look good 😊.

Reply:
> oooh Okay! @Abhijay007 I think you are thinking that the title need's to be part of the `app Bar` that's why you made that cancel button align to the title right! but actually, that was the purpose of the design that was not an alignment issue!
> 
> Adding the Cancel button to the `Material AppBar` itself makes it responsive so I don't think this will make any issue!
> 

yes

Reply:
> > And in all other pages of the site as well the title of the page is in `Appbar` along with goBack and navigate button :)
> 
> @Abhijay007 Actually it's not a mandatory thing to sync every route page with the same format. We can change the format for some routes like this section just the thing is it should look good 😊.

Ya ! as the horizontal instability issue for smaller devices is resolved I think it is fine to implement this 😊 .

Reply:
So @Abhijay007 you can revert the changes 😊

Reply:
> So @Abhijay007 you can revert the changes 😊

Can I revert the changes after knowing @Abhishek-kumar09 thoughts?

Reply:
> Can I revert the changes after knowing @Abhishek-kumar09 thoughts?

Sure no worries  :)

Reply:
Referral page is more like a paper where there is coupon code and glossy design, It is not like the usual page and more like a special page. 

I was comapring https://deploy-preview-296--edu-client.netlify.app/referralAndCoupons and
https://edu-client.netlify.app/referralAndCoupons

You have lowered the cross button and made the cross and title on same level, which is fine but initial design is looking better in terms of the purpose and theme though there are not much difference, and I may be talking garbage. 
Also the design was approved by few mentors so we can leave it as it is for now. Though it's not a big deal and thanks for your efforts.

Reply:
> Referral page is more like a paper where there is coupon code and glossy design, It is not like the usual page and more like a special page.
> 
> I was comapring https://deploy-preview-296--edu-client.netlify.app/referralAndCoupons and
> https://edu-client.netlify.app/referralAndCoupons
> 
> You have lowered the cross button and made the cross and title on same level, which is fine but initial design is looking better in terms of the purpose and theme though there are not much difference, and I may be talking garbage.
> Also the design was approved by few mentors so we can leave it as it is for now. Though it's not a big deal and thanks for your efforts.

ok, no problem I will revert the changes 😊 , thanks @adarsh-technocrat @Abhishek-kumar09 for suggestions.